services/minespace-web/src/tests/components/project/informationRequirementsTablePage/__snapshots__/InformationRequirementsTablePage.spec.tsx.snap

```diff
@@ -0,0 +1,17 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`InformationRequirementsTablePage renders properly 1`] = `
+<Provider
+  store={
+    Object {
+      "@@observable": [Function],
+      "dispatch": [Function],
+      "getState": [Function],
+      "replaceReducer": [Function],
+      "subscribe": [Function],
+    }
+  }
+>
+  <InformationRequirementsTablePage />
+</Provider>
```

Comment:
I'm wondering if we could do a full render of this to get a bit more usable snapshot?